front/app/components/IdeaButton/index.tsx

```diff
@@ -154,7 +154,7 @@ const IdeaButton = memo<Props & InjectedIntlProps>(
       authUser,
     });
 
-    const onClick = (event: React.FormEvent<HTMLButtonElement>) => {
+    const onClick = (event: React.MouseEvent) => {
```

Comment:
The previous type was not accurate as our Button is not always a part of a form. It is now changed to MouseEvent.